templates/todo/projects/nodejs-mongo/infra/applicationinsights.bicep

```diff
@@ -3,8 +3,10 @@ param location string
 param tags object
 param workspaceId string
 
+var abbrs = loadJsonContent('../../../../common/infra/abbreviations.json')
```

Comment:
Just need to add the following to your `repo.yaml`

```yaml
  rewrite:
    patterns:
      - "**/*.@(yml|yaml)"
      - "**/*.bicep"
```

See https://github.com/Azure/azure-dev/pull/162/files#diff-b0ec1cd3c68dbde790266bd1136ee50c4e1a8fd06aad87bf7e79daf97efba12e for an example.

Reply:
Got it.  Thanks!